### ISSUE TITLE
lr scheduler `LambdaLR` doesn't follow PyTorch's LRScheduler API

### ISSUE THREAD

**Issue by TorAP** [CompVis/latent-diffusion#319]
`pytorch_lightning.utilities.exceptions.MisconfigurationException: The provided lr scheduler `LambdaLR` doesn't follow PyTorch's LRScheduler API. You should override the `LightningModule.lr_scheduler_step` hook with your own logic if you are using a custom LR scheduler.`

Anyone else encountering this problem?

I have tried with torch 2.0 and 2.0.1

**Comment by omrastogi** [comment]
@TorAP how did you solve this problem
